vispy/scene/widgets/colorbar.py

```diff
@@ -133,6 +133,15 @@ def label(self, label):
         self._colorbar.label = label
 
     @property
+    def label_str(self):
+        return self._label_str
+
+    @label_str.setter
+    def label_str(self, label_str):
+        self._label_str = label_str
```

Comment:
Please change this also to `self._colorbar.label_str`.

Reply:
Will do as soon as I can.